vignettes/model_fitting.Rmd

```diff
@@ -373,3 +370,45 @@ sq <- SurvivalQuantities(
 )
 brierScore(sq)
 ```
+
+
+## Initial Values
+
+By default `jmpost` will set the initial values for all parameters to be a random value
+drawn from the prior distribution that has been shrunk towards the
+mean of said distribution e.g. for a `prior_normal(4, 2)`
```

Comment:
would be good to be able to control the amount of shrinkage with a package option